### DISCUSSION TITLE
Some Scripts are not "starting"

### DISCUSSION THREAD

**Discussion by Tolbag** [tteck/Proxmox#2699] [Help Needed] [answered]
Hey,
I have a new Server with Proxmox. My old Container and VM are restored and running.

However, some scripts do not work, they work on the old server.
e.g. the script for the CPU Governor or the Proxmox VE Post Install script.
I copy the command into the shell and press Enter, but nothing happens. On the old server it works as I said.

Can anyone help? Could it be because the time is not correct? I can't get this set.

![image](https://github.com/tteck/Proxmox/assets/64642996/d0ee7096-5c55-4e4e-b8b3-f35a04c6199b)

Regards
tolbag

**Comment by tteck** [comment, marked as answer]
🌐 It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by Tolbag** [reply]
Great! Thank you!! My DNS was 127.0.0.1 ...

**Reply by brad-thomson** [reply]
I'm having a similar issue after switching from my tp-link router to an OPNsense vm. I checked proxmox dns and my nameservers are set to 1.1.1.1 and 9.9.9.9, but still the exact same symptoms as Tolbag. I'm not that far into my homelab journey but wonder if I could get some help?

**Reply by mrodilcu** [reply]
Same issue, and same DNS server as @GlowDrops. Any ideas? Thx
